docs/concepts/ShieldedAddress.md

```diff
@@ -0,0 +1,8 @@
+# Shielded Addresses
+
+One essential concept in MantaPay is **shielded addresses**. A shielded address is a reusable public address for your private assets in Manta. It has the following properties:
```

Comment:
I would remove 'public' here; I think it is a little confusing to say that a 'public address' holds private assets without context. I think the explanation below (2.) is enough